code/items/equipments/entities/C4Entity.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Sandbox;
+using Sandbox.UI;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_c4_ent"), Hammer.Skip]
+    public partial class C4Entity : Prop, IUse
+    {
+        private string ModelPath => "models/entities/c4.vmdl";
+
+        [Net]
+        public int AttachedBone { get; set; } = -1;
+
+        [Net]
+        public bool IsArmed { get; set; } = false;
+
+        //Timer display on C4 entity.
+        private WorldPanel Display;
+        private bool CreatedDisplay = false;
```

Comment:
Could avoid the comment, and just make the variable name clearer.

`BOMB_DISPLAY`